examples/orders/list.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * @docs https://docs.mollie.com/reference/v2/orders-api/list-orders
+ */
+const mollie = require('@mollie/api-client');
+
+const mollieClient = mollie({ apiKey: 'test_dHar4XY7LxsDOtmnkVtjNVWXLSlXsM' });
+
+(async () => {
+  try {
+    const mostRecentOrders = await mollieClient.orders.page();
```

Comment:
I've tried here to make it look more like the other clients and the documentation. This is why there is both a `.page` and `.list` alias. If that is too confusing, then it might be better to remove the aliases.

Reply:
I like the aliases! That part is great. But I think we should be consistent with naming in our examples, as this might confuse new devs trying out this library. Since we use ".all" in other examples as well, this might make more sense to use in this example as well.

Reply:
Using `.all` in all examples now.